db/models/applications.js

```diff
@@ -87,6 +87,18 @@ class PendingApplication extends Model {
             order: [['createdAt', 'DESC']]
         });
     }
+    static findApplicationsByExpiry() {
+        const startDate = moment().toDate();
+        const endDate = moment().add('1', 'months').toDate();
+
+        return this.findAll({
+            where: {
+                expiresAt: {
+                    [Op.between]: [startDate, endDate]
+                }
+            }
+        });
+    }
```

Comment:
Not sure if this is helpful but the following SQL query will give you the number of days until each application expires:

`SELECT DATEDIFF(expiresAt, CURDATE()) AS daysTilExpiry FROM PendingApplications`

... though I'm not sure how to translate that into Sequelize syntax...

Reply:
Yup, I used the same query and the syntax I used is the sequelize translation of it. Though,  unfortunately I couldn't find any confirmation through their docs. Had to dig a lot to find the date compare example in their issues list.